pkg/api/customization/cluster/actions_cis.go

```diff
@@ -86,35 +86,19 @@ func (a ActionHandler) runCisScan(actionName string, action *types.Action, apiCo
 		}
 	}
 
-	newCisScan := cis.NewCisScan(cluster, cisScanConfig)
-	cisScan, err := a.ClusterScanClient.Create(newCisScan)
+	isManual := true
+	cisScan, err := cis.LaunchScan(
+		isManual,
+		cisScanConfig,
+		cluster,
+		a.ClusterClient,
+		a.ClusterScanClient,
+		RetryIntervalInMilliseconds,
```

Comment:
knitpick: do we really need to pass in these two retry parameters? are they ever not the same thing?

Reply:
Will clean up.